css/style.css

```diff
@@ -46,7 +53,69 @@ li {
   display: inline;
 }
 
-#storeTable {
-  border: 5px;
-  border-color: green;
-}
\ No newline at end of file
+/* Below this is for the index.html page */
+
+
+p {
+  margin-left: auto;
+  margin-right: auto;
+  width: 70%;
+}
+
+#headerfish {
+  display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 40%;
+}
+
+h1 {
+  text-align: center;
+  font-size: 40px;
+  text-transform: capitalize;
+  color: black;
+}
+
+h2 {
+  font-weight: bolder;
+}
+
+#Slogan {
+  margin-left: auto;
+  margin-right: auto;
+  width: 45%;
+  font-size: 15px;
+}
+
+section {
+  margin: 15px;
+  padding: 30px;
+  text-align: center;
+  position: relative;
+  overflow: hidden;
+}
+
+#sellout {
+  overflow: hidden;
+  height: 300px;
+  width: 300px;
+}
+
+#sellout img {
+  width: 400px;
+  height: 300px;
+  margin: -175px -175px -175px -175px;
```

Comment:
You should try and refrain from using negative margins, most of the time it is not necessary.